addons/danger/functions/fnc_leaderArtillery.sqf

```diff
@@ -33,16 +33,14 @@ _artillery = _artillery select {
 };
 
 // exit on no ready artillery
-if (_artillery isEqualTo []) exitWith {if (GVAR(debug_functions)) then {format ["%1 Artillery failed -- no available artillery", side _unit] call FUNC(debugLog);}};
+if (_artillery isEqualTo []) exitWith {if (GVAR(debug_functions)) then {format ["%1 Artillery failed -- no available artillery in rage of Target", side _unit] call FUNC(debugLog);}};
```

Comment:
"no available artillery in raNge of Target"

Reply:
i just copied it from the one blow